gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Let's update the Gradle version and regenerate the Gradle Wrapper and commit the Jar.

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.11.1-bin.zip
```